### PR TITLE
Docs: Document jaeger tracing instrumentation

### PR DESCRIPTION
**Pull request by Dieterbe**
jaeger instrumentation was introduced in https://github.com/grafana/grafana/pull/9213 but it was not documented yet apparently :?
I didn't run any tests!

cc @bergquist , @robbymilo

### CODE REVIEW

File: docs/sources/administration/jaeger-instrumentation.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+keywords = ["grafana", "jaeger", "tracing"]
+type = "docs"
+[menu.docs]
+parent = "admin"
+weight = 9
++++
+
+# Jaeger instrumentation
+
+Grafana supports [jaeger tracing](https://www.jaegertracing.io/).
```
Comment (oddlittlebird) [inline docs/sources/administration/jaeger-instrumentation.md]:
```suggestion
Grafana supports [Jaeger tracing](https://www.jaegertracing.io/).
```

File: docs/sources/administration/jaeger-instrumentation.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Grafana can emit jaeger traces for its http api endpoints as well as propagate jaeger trace information to datasources.
```
Comment (oddlittlebird) [inline docs/sources/administration/jaeger-instrumentation.md]:
```suggestion
Grafana can emit Jaeger traces for its HTTP API endpoints and propagate Jaeger trace information to data sources. 
```

Comment (oddlittlebird) [reply docs/sources/administration/jaeger-instrumentation.md]: Unclear if "its HTTP API endpoints" are Grafana's or Jaeger's. Please clarify.

File: docs/sources/administration/jaeger-instrumentation.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+parent = "admin"
+weight = 9
++++
+
+# Jaeger instrumentation
+
+Grafana supports [jaeger tracing](https://www.jaegertracing.io/).
+
+Grafana can emit jaeger traces for its http api endpoints as well as propagate jaeger trace information to datasources. 
+All HTTP endpoints are logged evenly. (annotations, dashboard, tags, etc)
```
Comment (oddlittlebird) [inline docs/sources/administration/jaeger-instrumentation.md]:
```suggestion
All HTTP endpoints are logged evenly (annotations, dashboard, tags, and so on).
```

File: docs/sources/administration/jaeger-instrumentation.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+When a trace ID is propagated, it is reported with operation 'HTTP /datasources/proxy/:id/*'.
+
+Jaeger tracing is enabled in the [tracing.jaeger] section in your [grafana.ini](https://grafana.com/docs/grafana/latest/installation/configuration/#tracing-jaeger) config file.
```
Comment (oddlittlebird) [inline docs/sources/administration/jaeger-instrumentation.md]:
```suggestion
Refer to [Configuration](https://grafana.com/docs/grafana/latest/installation/configuration/#tracing-jaeger) for information about enabling Jaeger tracing.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
This is a good start, but it is a bit abrupt. As a user, I have questions.

**Why do I care about this?** Okay, if I use Jaeger, I guess I care. But if I don't know what Jaeger is? Perhaps give a sentence or two so the reader can get some context and decide if they want to know more.

**How do I do this?** I infer that the user needs to update grafana.ini, but you also mention API endpoints and output. 

So, here's what this should have:
Concept - Explain what you are talking about. Introduction. 
Task - What tasks is the user going to do? Enable Jaeger tracing? View trace results? These should be step-by-step instructions.
Reference - If applicable, this would be things like a list of API endpoints, buttons or fields on a panel in a reference table, defined terms, and so on.

File: docs/sources/administration/jaeger-instrumentation.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+title = "Jaeger instrumentation"
+description = "Jaeger traces emitted and propagation by Grafana"
+keywords = ["grafana", "jaeger", "tracing"]
+type = "docs"
+[menu.docs]
+parent = "admin"
+weight = 9
++++
+
+# Jaeger instrumentation
```
Comment (oddlittlebird) [inline docs/sources/administration/jaeger-instrumentation.md]: When we get a better handle on the final shape of the documentation, we should revisit this title. I suspect we can come up with something more descriptive and accurate.

**Comment by Dieterbe** [comment]
Can someone on the Grafana team continue this work ?

**Comment by oddlittlebird** [comment]
@daniellee, see question from @Dieterbe

**Comment by marefr** [comment]
Think zipkin is supported as well when configured using zipkin propagation format, https://grafana.com/docs/grafana/latest/installation/configuration/#zipkin-propagation

**Comment by bergquist** [comment]
Yea we should rename this tracing support instead even thou only the jaeger client is implemented.

**Comment by Dieterbe** [comment]
Don't let perfect be the enemy of good.
We have something, let's get it merged. We can always improve later.

**Comment by Dieterbe** [comment]
@daniellee said he'd take care of this :)

**Comment by Dieterbe** [comment]
this is starting to get painful. :'(

**Comment by Dieterbe** [comment]
:hourglass: :hourglass: :hourglass: :hourglass: 

> Don't let perfect be the enemy of good.
> We have something, let's get it merged. We can always improve later.

**Review by oddlittlebird — APPROVED** [review]

**Pull request merged**